public/manifest.json

```diff
@@ -0,0 +1,511 @@
+      "sizes": "72x72"
+    },
+    {
+      "src": "icons/windows11/Square44x44Logo.altform-lightunplated_targetsize-80.png",
+      "sizes": "80x80"
+    },
+    {
+      "src": "icons/windows11/Square44x44Logo.altform-lightunplated_targetsize-96.png",
+      "sizes": "96x96"
+    },
+    {
+      "src": "icons/windows11/Square44x44Logo.altform-lightunplated_targetsize-256.png",
+      "sizes": "256x256"
+    },
+    {
+      "src": "icons/android/android-launchericon-512-512.png",
+      "sizes": "512x512"
+    },
+    {
+      "src": "icons/android/android-launchericon-192-192.png",
+      "sizes": "192x192"
+    },
+    {
+      "src": "icons/android/android-launchericon-144-144.png",
+      "sizes": "144x144"
+    },
+    {
+      "src": "icons/android/android-launchericon-96-96.png",
+      "sizes": "96x96"
+    },
+    {
+      "src": "icons/android/android-launchericon-72-72.png",
+      "sizes": "72x72"
+    },
+    {
+      "src": "icons/android/android-launchericon-48-48.png",
+      "sizes": "48x48"
+    },
+    {
+      "src": "icons/ios/16.png",
+      "sizes": "16x16"
+    },
+    {
+      "src": "icons/ios/20.png",
+      "sizes": "20x20"
+    },
+    {
+      "src": "icons/ios/29.png",
+      "sizes": "29x29"
+    },
+    {
+      "src": "icons/ios/32.png",
+      "sizes": "32x32"
+    },
+    {
+      "src": "icons/ios/40.png",
+      "sizes": "40x40"
+    },
+    {
+      "src": "icons/ios/50.png",
+      "sizes": "50x50"
+    },
+    {
+      "src": "icons/ios/57.png",
+      "sizes": "57x57"
+    },
+    {
+      "src": "icons/ios/58.png",
+      "sizes": "58x58"
+    },
+    {
+      "src": "icons/ios/60.png",
+      "sizes": "60x60"
+    },
+    {
+      "src": "icons/ios/64.png",
+      "sizes": "64x64"
+    },
+    {
+      "src": "icons/ios/72.png",
+      "sizes": "72x72"
+    },
+    {
+      "src": "icons/ios/76.png",
+      "sizes": "76x76"
+    },
+    {
+      "src": "icons/ios/80.png",
+      "sizes": "80x80"
+    },
+    {
+      "src": "icons/ios/87.png",
+      "sizes": "87x87"
+    },
+    {
+      "src": "icons/ios/100.png",
+      "sizes": "100x100"
+    },
+    {
+      "src": "icons/ios/114.png",
+      "sizes": "114x114"
+    },
+    {
+      "src": "icons/ios/120.png",
+      "sizes": "120x120"
+    },
+    {
+      "src": "icons/ios/128.png",
+      "sizes": "128x128"
+    },
+    {
+      "src": "icons/ios/144.png",
+      "sizes": "144x144"
+    },
+    {
+      "src": "icons/ios/152.png",
+      "sizes": "152x152"
+    },
+    {
+      "src": "icons/ios/167.png",
+      "sizes": "167x167"
+    },
+    {
+      "src": "icons/ios/180.png",
+      "sizes": "180x180"
+    },
+    {
+      "src": "icons/ios/192.png",
+      "sizes": "192x192"
+    },
+    {
+      "src": "icons/ios/256.png",
+      "sizes": "256x256"
+    },
+    {
+      "src": "icons/ios/512.png",
+      "sizes": "512x512"
+    },
+    {
+      "src": "icons/ios/1024.png",
+      "sizes": "1024x1024"
+    }
+  ]
```

Comment:
**Question:** It's a dang lot of icons, I hope you used some kind of tool for them in case they ever change? Do we even need all of those variants and do PWAs not support SVGs for favicons?


**Question:** Also, do we actually expect anyone to install the nova website as a PWA, doesn't really make sense to me, but it's not like it fundamentally changes anything so it doesn't hurt to add support for it.

Reply:
idea i like it in the future the ability to send notifications or have the ability to open a piece of code in docs to try to find references

Reply:
or even just offline blog post reading